invenio_oauthclient/views/rest.py

```diff
@@ -0,0 +1,161 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think there's still a bit of work here between client.py and rest.py.

Same pattern repeats in all functions:

![Screenshot 2020-05-12 at 16 08 25](https://user-images.githubusercontent.com/1698163/81702109-2916e800-946b-11ea-96ad-3ccea58b2034.png)